CODEOWNERS

```diff
@@ -1 +1 @@
-* @yorinasub17  @rhoboat  @autero1
+* @autero1 @yorinasub17
```

Comment:
```suggestion
```

Reply:
I imagine both of them are also no longer code owners here?

Reply:
Indeed - I think we can probably just delete this file - or just put me as CODEOWNER?

Reply:
Either option sounds good to me!